src/ui/screens/home/controller/ConnectionTimer.qml

```diff
@@ -12,6 +12,9 @@ RowLayout {
     property real defaultWidth: MZTheme.theme.fontSize * 0.25
     property real narrowWidth: MZTheme.theme.fontSize * 0.05
     property var narrowCharacters: [" ", "\t", "\n", ":"]
+    property bool ignoreForAccessibility: false
+
+    Accessible.ignored: ignoreForAccessibility
```

Comment:
why do we need to specifically ignore the `RowLayout` instead of just the `Text` component?

Reply:
I was trying to the keep the old behavior:
          ConnectionTimer {
            id: connectionTime
            Accessible.ignored: true
          }

But it isn't needed. Will remove this.